pkg/store/filesystem/bundles.go

```diff
@@ -23,7 +23,7 @@ func (s Store) ListBundles(bundleType proto.Bundle_Type) ([]*proto.Bundle, error
 	bundlesFolder := s.buildFolderPathForBundles(false)
 	bundleFiles, err := ioutil.ReadDir(bundlesFolder)
 	if err != nil {
-		return []*proto.Bundle{}, nil
+		return []*proto.Bundle{}, merrors.FromError(err)
```

Comment:
replace with `return []*proto.Bundle{}, err`

Reply:
I'd restrain from returning an error here, as callers would check for it. Either return an empty bundle or an error, not both. I'd leave it as it was.